src/Pact/Eval.hs

```diff
@@ -659,8 +659,9 @@ enforceAcyclic
   -> Eval e [(Term (Either l r), key, [key])]
 enforceAcyclic info cs = forM cs $ \c -> case c of
   AcyclicSCC v -> return v
-  CyclicSCC vs -> do
-    let i = if null vs then info else _tInfo $ view _1 $ head vs
+  CyclicSCC [] -> error "Expected nonempty list"
```

Comment:
The previous impl proceeded as usual binding `i` to `info` in this case if my brain is symbolic-executing the code right. Although I'm not sure if that makes sense given the semantics of this branch.